tabbycat/api/serializers.py

```diff
@@ -117,10 +117,10 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Round
-        fields = '__all__'
+        exclude = ('tournament',)
 
     def validate(self, data):
-        if (data['break_category'] is None) == (data['stage'] == Round.STAGE_ELIMINATION):
+        if (data.get('break_category') is None) == (data.get('stage') == Round.STAGE_ELIMINATION):
```

Comment:
Ah, this "XNOR". Haha, it's a bit obscure but you're right, I can't think of a cleaner way, I'd probably do the same.